.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,16 @@
-_Please note that while we are transitioning from travis-ci to AWS CodeBuld, some tests are run on each platform. Non-AWS contributors will temporarily be unable to see CodeBuild results. We apologize for the inconvenience._
+_Please note that while we are transitioning from travis-ci to AWS CodeBuild, some tests are run on each platform. Non-AWS contributors will temporarily be unable to see CodeBuild results. We apologize for the inconvenience._
 
-**Issue # (if available):** 
+### Summary of changes: 
 
-**Description of changes:** 
+Describe s2n’s current behavior and how your code changes that behavior. If there are no issues this pr is resolving, explain why this change is necessary.
+### Call-outs:
 
+Address any potentially confusing code. Is there code added that needs to be cleaned up later? Is there code that is missing because it’s still in development? 
```

Comment:
One of the questions I'd love to thought about in every PRs is risk. Whether risk should be part of call out or its own section is less important to the thought exercise. Is the change risky? Are there potential breaking changes? Is there a potential performance degradation? Is there potentially for something to be abused, misused, misconfigured...

Reply:
We should probably revisit our design process to address this. The PR is too late to find out the performance will be degraded, or that configuration isn't easy for users. We want to know about those types of concerns before code is written.

Reply:
I agree with Ryan here; if we're committing risky code we should have already had a discussion about it in a design or doc review.